src/lib/swapper/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -105,7 +73,7 @@ export async function getTradeQuote(
     toTokenAddress: buyLifiToken.address,
     fromAddress: receiveAddress,
     toAddress: receiveAddress,
-    fromAmount: thresholdedAmountCryptoLifi,
+    fromAmount: fromAmountLifiCryptoPrecision.toString(),
```

Comment:
@gomesalexandre is that _really_ any safer if we are not passing any args?

Reply:
Yeah, when called without any arguments, `.toFixed()` might not *necessarily* be safer, though that's what we use when dealing with precision amounts for *consistency*. It tends to deal with exponents/scientific notation better and *consistently* outputs fixed-point notation amounts. 

Now, there's actually another issue here and that's the naming:

<img width="1408" alt="image" src="https://user-images.githubusercontent.com/17035424/231717257-4e497e64-a19d-4513-8a87-4b08511fe8ac.png">

This doesn't actually return a `lifiCryptoPrecision` but a `lifiCryptoBaseUnit` i.e an amount in the base unit of the `sellLifiToken`

Reply:
Disregard me again - `RoutesRequest` is Li.Fi SDK's payload, so we obviously can't rename this.

- The `base unit` terminology still stands however, confirmed with morning wood the precision vernacular was a mistake and we should send base unit `fromAmount`s in the payload - double-checked against their docs.
- It looks like the `lifi` vernacular could be removed altogether as this is really the asset's base unit, though confirmed the reason why we get a "lifi asset's precision" is because we want to paranoia guard against their asset's precision possibly being different from ours, so `lifi` vernacular is actually valid.

Reply:
Disregard.Me.S01E03.FRENCH.1080p.WEB.H264-FOX - confirmed against https://apidocs.li.fi/reference/get_tokens-1 the precision is actually correct for LiFi assets, which would be a concern if it wasn't.

Removed the `lifi` terminology in this PR (9f8a718be) and used the correct `baseUnit` vernacular for now - will do a follow-up to remove the redundant `lifiPrecision` stuff